File: src/main/java/com/tickets4u/model/Evento.java
package com.tickets4u.model;

import jakarta.persistence.*;
import java.time.LocalDateTime;

@Entity
public class Evento {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @OneToOne
    @JoinColumn(name="usuario_id")
    private Long id_admin;
    private String nombre;
    private String descripcion;
    private LocalDateTime fechaInicio;
    private LocalDateTime fechaFin;
    private String direccion;
    private String foto;
    private String qr;


}

File: frontend/android/app/src/main/java/com/example/tickets4u/Evento.kt
package com.example.tickets4u

import java.time.LocalDateTime

data class Evento(
    val id: Int,
    val id_admin: Int,
    val nombre: String,
    val descripcion: String,
    val fecha_inicio: LocalDateTime,
    val fecha_fin: LocalDateTime,
    val direccion: String,
    val foto: String,   // imagen del evento
    val qr: String? = null // QR opcional
)
